### PR TITLE
✨ [FEAT] : Add banner for NFTs designed for Stax

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Add banner for NFTs designed for Stax in : 
- CustomLock Screen drawer
- Collection
- Gallery
- Nft Detail viewer

Screenshot 2024-05-23 at 15 28 47
Screenshot 2024-05-23 at 15 28 16
Screenshot 2024-05-23 at 15 28 12
Screenshot 2024-05-23 at 15 27 51

### ❓ Context

- **JIRA or GitHub link**: [LIVE-12560] 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-12560]: https://ledgerhq.atlassian.net/browse/LIVE-12560?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sshmaxime — APPROVED** [review]
🚀

File: apps/ledger-live-desktop/src/renderer/components/Nft/DesignedForStax.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  const { colors } = useTheme();
+  return (
+    <Flex px={size === "small" ? 1 : 2}>
+      <IconsLegacy.LedgerLogoMedium
+        size={size === "medium" ? 24 : size === "large" ? 36 : 16}
+        color={hideBackground ? colors.neutral.c100 : colors.constant.white}
+      />
+    </Flex>
+  );
+};
```
Comment (KVNLS) [inline apps/ledger-live-desktop/src/renderer/components/Nft/DesignedForStax.tsx:32]:
```suggestion
const LedgerLogo = ({ size, hideBackground }: DesignedForStaxTextProps) => {
  const { colors } = useTheme();
  return (
    <Flex px={size === "small" ? 1 : 2}>
      <IconsLegacy.LedgerLogoMedium
        size={size === "medium" ? 24 : size === "large" ? 36 : 16}
        color={hideBackground ? colors.neutral.c100 : colors.constant.white}
      />
    </Flex>
  );
};
```

If children is not used, why do we not remove it from props?

File: apps/ledger-live-desktop/src/renderer/components/Nft/DesignedForStax.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    px={size === "small" ? 0 : 1}
+    variant="tiny"
+    fontSize={size === "medium" ? 14 : size === "large" ? 20 : 12}
+    textTransform="uppercase"
+    textAlign="center"
+    color={hideBackground ? "neutral.c100" : "constant.white"}
+  >
+    {children}
+  </Text>
+);
```
Comment (KVNLS) [inline apps/ledger-live-desktop/src/renderer/components/Nft/DesignedForStax.tsx:51]: As TagText is only used here and never with children, why do we use the children?

File: apps/ledger-live-desktop/src/renderer/components/Nft/DesignedForStax.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  size,
+  hideBackground,
+}: {
+  children?: string;
+} & DesignedForStaxTextProps) => {
+  const { colors } = useTheme();
+  return (
+    <Flex px={size === "small" ? 1 : 2}>
+      <IconsLegacy.LedgerLogoMedium
+        size={size === "medium" ? 24 : size === "large" ? 36 : 16}
```
Comment (KVNLS) [inline apps/ledger-live-desktop/src/renderer/components/Nft/DesignedForStax.tsx:27]: We have different font sizes used here, can't we use the ones from our theme? Is there any Figma that defines these font size values?

**Review by KVNLS — CHANGES_REQUESTED** [review]

**Review by ofreyssinet-ledger — APPROVED** [review]
good for device scope (custom lock screen component)

**Pull request closed without merging**